### PR TITLE
Update macOS archive installation with ARM64 for 22/3

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *Add ARM 64-bit*
 - *Update package names to temporary ones. Note that the final package names will be updated in another PR when the packages are ready to release*

### Tickets:
 - *[CVS-98906](https://jira.devtools.intel.com/browse/CVS-98906)*

### CODE REVIEW

**Review by totoka-intel — APPROVED** [review]

File: docs/install_guides/installing-openvino-from-archive-macos.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNo
 
 .. tab:: Operating Systems
 
-  macOS 10.15, 11, 12, and 13 versions with the x86 architecture, 64 bits
+  macOS 10.15, 11, 12, and 13, x86, 64-bit
+  macOS 10.15, 11, 12, and 13, ARM, 64-bit
```
Comment (yatarkan) [inline docs/install_guides/installing-openvino-from-archive-macos.md:20]: I suppose this line with OS requirements contradicts note above (L11, `> **NOTE**: The Intel® Distribution of OpenVINO™ toolkit is supported on macOS version 10.15 with Intel® processor-based machines.`)
Should this note also be updated or removed?

Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-macos.md]: Thanks for pointing it out. I noticed it too. Removed the specific mac OS version in the note. :)

File: docs/install_guides/installing-openvino-from-archive-macos.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 
 Installing OpenVINO Runtime from archive files is recommended for C++ developers. If you are working with Python, the PyPI package has everything needed for Python development and deployment on CPU and GPUs. Visit the [Install OpenVINO from PyPI](installing-openvino-pip.md) page for instructions on how to install OpenVINO Runtime for Python using PyPI.
 
 See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes) for more information on updates in the latest release.
 
 > **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can be installed via [pypi.org](https://pypi.org/project/openvino-dev/) only.
 
-> **NOTE**: The Intel® Distribution of OpenVINO™ toolkit is supported on macOS version 10.15 with Intel® processor-based machines.
+> **NOTE**: Intel® Distribution of OpenVINO™ toolkit is supported on macOS with Intel® processor-based machines.
```
Comment (yatarkan) [inline docs/install_guides/installing-openvino-from-archive-macos.md:11]: As far as I see, wording `with Intel® processor-based machines` tells user that nothing except Intel-based machines is supported. But below we also mention ARM in the table with Operating Systems: `* macOS 10.15, 11, 12, and 13, ARM, 64-bit`.

File: docs/install_guides/installing-openvino-from-archive-macos.md [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
 
 @sphinxdirective
 
 .. tab:: Operating Systems
 
-  macOS 10.15, 11, 12, and 13 versions with the x86 architecture, 64 bits
+   * macOS 10.15, 11, 12, and 13, x86, 64-bit
+   * macOS 10.15, 11, 12, and 13, ARM, 64-bit
   
   .. note::
```
Comment (yatarkan) [inline docs/install_guides/installing-openvino-from-archive-macos.md:23]: Question regarding the following note:
```
Only macOS 10.15 is fully validated with OpenVINO. Other versions depend on the compatibility of macOS.
```
Are macOS versions 11, 12, 13 are validated and missed in this note?
If not - do we officially support macOS 11, 12 and 13 without validation?

Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-macos.md:23]: Yes, that was suggested by Helena and Ilya Lavrenov, macOS 11, 12 and 13 are supported but not validated.

File: docs/install_guides/installing-openvino-from-archive-macos.md (hunk 2 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-macos.md:11]: OK I see your point. Then this note is not true anymore. I'll remove it.

File: docs/install_guides/installing-openvino-from-archive-macos.md [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
-
 ## System Requirements
 
 @sphinxdirective
 
 .. tab:: Operating Systems
 
-  macOS 10.15, 11, 12, and 13 versions with the x86 architecture, 64 bits
+   * macOS 10.15, 11, 12, and 13, x86, 64-bit
+   * macOS 10.15, 11, 12, and 13, ARM, 64-bit
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-from-archive-macos.md:18]: 10.15 is not valid for arm. arm is supported only since macOS 11

Comment (ilya-lavrenov) [reply docs/install_guides/installing-openvino-from-archive-macos.md:18]: Hardware tab is also now valid only for x86, for arm we need a different hardware list - Apple M1 / M2 processors

**Pull request merged**